contracts/nois-proxy/src/contract.rs

```diff
@@ -44,6 +45,7 @@ pub fn instantiate(
         test_mode,
         callback_gas_limit,
         payment: None,
+        randomness_price,
```

Comment:
This should be updatable after instantiation. But the proxy has no manager right now.

Reply:
I think we should work on some good docs on how to deploy and how to maintain proxies. 
Adding a manager now and deploying it means that Nois we'll be in charge of updating the prices for the whole fleet and in my opinion this is not a business Nois should maintain long term. So we can make a simple and workable proxy witha fixed price then chains or dapps will have to maintain their own proxies the way they like it.

Reply:
So I suggest we leave the price immutable at instantiation for now. worse case scenario dapps can always prefill their payment contract on Nois so the whole process of proxy payment is optional anyways.

Reply:
okay, so if maintaining the proxies is not our job long term, then we should make it super easy for others to do that. So we could hardcode the NOIS denom here and let the proxy as for the price automatically via IBC.

Reply:
Oh, sorry, hardcode was wrong. I meant here in instantiate we configure the denom.

I'll push a change in a few minites that implements it, then we can discuss it in detail.

Reply:
Yeah I think so to. It is best practive to only have one IBC representation of a token and maintain that long term. I.e. NOIS in Juno should alsways be ibc/1D9E14A1F00613ED39E4B8A8763A20C9BE5B5EA0198F2FE47EAE43CD91A0137B on channel channel-225